src/coord/axisCommonTypes.ts

```diff
@@ -219,6 +220,8 @@ interface AxisLabelBaseOption extends Omit<TextCommonOption, 'color'> {
     // Whether axisLabel is inside the grid or outside the grid.
     inside?: boolean,
     rotate?: number,
+    autoRotate?: boolean | [number],
+    minDistance?: number,
```

Comment:
`minDistance` is the minimal-allowed distance between the labels before they are rotated/collapsed.